pom.xml

```
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>life.qbic</groupId>
 	<artifactId>sampletracking</artifactId>
-	<version>1.1.0</version>
+	<version>1.2.0</version>
 	<properties>
```

Please make sure that the release number fits the planned release 🔢 
```suggestion
	<version>1.2.1</version>
	<properties>
```